patches/jsii-pacmak+1.71.0.patch

```diff
@@ -0,0 +1,4180 @@
+diff --git a/node_modules/jsii-pacmak/bin/jsii-pacmak.js b/node_modules/jsii-pacmak/bin/jsii-pacmak.js
+index 96120e9..cb54420 100644
```

Comment:
weird, the diff shouldn't be this big

Reply:
You might've had 1.68.0 installed at the time of running the patch-package script? 🤔

Reply:
Yeah, patch-package took the package-lock.json file instead of the yarn.lock